frontend/packages/kubevirt-plugin/integration-tests-cypress/view/vm.ts

```diff
@@ -8,11 +8,16 @@ import { wizard } from './wizard';
 
 export const waitForStatus = (status: string, vmData?: VirtualMachineData, timeout?: number) => {
   const timeOut = timeout || VM_ACTION_TIMEOUT.VM_IMPORT;
-  cy.contains(detailsTab.vmStatus, status, { timeout: timeOut }).should('exist');
   if (status === VM_STATUS.Running) {
     const { name, namespace } = vmData;
     cy.waitForLoginPrompt(name, namespace);
   }
+
+  if (status === VM_STATUS.Starting) {
```

Comment:
a. https://github.com/kubevirt/user-guide/pull/439/files
maybe we may report starting when real status is provisioning ?
we should look at the logic for reporting "starting" and see if in some cases we should report "provisioning" instead ?

b. can we make a new method 
`export const waitForStatuses = (statuses: string[], vmData?: VirtualMachineData, timeout?: number) => {`
and use it when we can continue if we see one of the correct statuses ?
e.g.
continue to next step if vm status is in `["starting", "provisioning"]` acceptable statuses list ...